chartmogul/api/customer.py

```diff
@@ -72,6 +73,18 @@ def make(self, data, **kwargs):
 
     _schema = _Schema(unknown=EXCLUDE)
 
+    @classmethod
+    def contacts(cls, config, **kwargs):
+        path = cls._preparePath('all', '/customers/{uuid}/contacts', kwargs)
+
+        return Contact._request(config, 'all', 'get', path, **kwargs)
+
+    @classmethod
+    def createContact(cls, config, **kwargs):
+        path = cls._preparePath('create', '/customers/{uuid}/contacts', kwargs)
+
+        return Contact._request(config, 'create', 'post', path, **kwargs)
```

Comment:
I can't seem to do it with `Resource._method` because it will load the response using the `Customer` class instead of `Contact`. Therefore I used `Contact._request` and extracted some logic in the request into `._preparePath` to avoid repetition